operations/services.md

```diff
@@ -251,75 +235,74 @@ Adds availability blocks which are used to group related [Availability update](#
+| `StartUtc` | string | required, max length 3 months | Start of the interval in UTC timezone in ISO 8601 format. |
+| `EndUtc` | string | required, max length 3 months | End of the interval in UTC timezone in ISO 8601 format. |
 
 ### Response
 
 ```javascript
 {
-    "AvailabilityBlocks": [
+    "CategoryAvailabilities": [
         {
-            "Id": "e5a4654a4a94-86da-4f96-9efc-bd26d8db",
-            "ServiceId": "bd26d8db-86da-4f96-9efc-e5a4654a4a94",
-            "StartUtc": "2020-10-05T00:00:00Z",
-            "EndUtc": "2020-10-06T00:00:00Z",
-            "ReleasedUtc": "2020-10-04T00:00:00Z",
+            "Availabilities": [ 6, 7, 5 ],
+            "Adjustments ": [ 0, 1, -1 ],
+            "CategoryId": "773d5e42-de1e-43a0-9ce6-f940faf2303f"
         },
         {
-            "Id": "aaaa654a4a94-4f96-9efc-86da-bd26d8db",
-            "ServiceId": "bd26d8db-86da-4f96-9efc-e5a4654a4a94",
-            "StartUtc": "2020-11-05T00:00:00Z",
-            "EndUtc": "2020-11-06T00:00:00Z",
-            "ReleasedUtc": "2020-11-04T00:00:00Z",
+            "Availabilities": [ 7, 7, 7 ],
+            "Adjustments ": [ 1, 0, -1 ],
+            "CategoryId": "a0a7a5c5-c4ef-494a-8b34-6cca97629076"
         }
+    ],
+    "DatesUtc": [
+        "2016-12-31T23:00:00Z",
+        "2017-01-01T23:00:00Z",
+        "2017-01-02T23:00:00Z"
     ]
 }
 ```
 
 | Property | Type | Contract | Description |
 | --- | --- | --- | --- |
-| `AvailabilityBlocks` | array of [Availability blocks](#availability-block) | required | Availability blocks. |
+| `CategoryAvailabilities` | array of [Resource category availability](#resource-category-availability) | required | Resource category availabilities. |
+| `DatesUtc` | array of string | required | Covered dates in UTC timezone in ISO 8601 format. |
 
-#### Availability block
+#### Resource category availability
 
 | Property | Type | Contract | Description |
 | --- | --- | --- | --- |
-| `Id` | string | required | Unique identifier of the availability block. |
-| `ServiceId` | string | required | Unique identifier of the [Service](#service) the block is assigned to. |
-| `StartUtc` | string | required | Start of the interval in UTC timezone in ISO 8601 format. |
-| `EndUtc` | string | required | End of the interval in UTC timezone in ISO 8601 format. |
-| `ReleasedUtc` | string | required | Date when the [Availability update](#availability-update)s linked to the block will not reduce public availability in UTC timezone in ISO 8601 format. |
+| `CategoryId` | string | required | Unique identifier of the [Resource category](enterprises.md#resource-category). |
+| `Availabilities` | array of number | required | Absolute availabilities of the resource category in the covered dates. |
+| `Adjustments` | array of number | required | Relative availability adjustments set for resource category in the covered dates. |
```

Comment:
Will this now return sum of all adjustments on that date? I guess it should so that it's backwards compatible. But then I'd clarify it here - that the number is sum of all adjusments for the category on that day - general adjustments plus adjustments in all blocks.